source/dem/floating_grid.cc

```diff
@@ -17,11 +17,17 @@ FloatingGrid<dim, spacedim>::FloatingGrid(
 {
   if (floating_grid_parameters.mesh.file_name != "none")
```

Comment:
I don't think this floating grid is actually used anywhere now or am I wrong? The current floating grids are stored into the Rigid solid object in core...This whole thing should be deleted, am I wrong @shahabgol ??